races.py

```diff
@@ -0,0 +1,125 @@
+from bs4 import BeautifulSoup as soup 
+from urllib.request import urlopen as uReq
+import win_unicode_console
+#I need import this because I have s problem with encoding in my notebook
+win_unicode_console.enable()
+#I need import this because I have s problem with encoding in my notebook
+
+
+page_list = list(range(0, 91))
+
+for p in page_list:
+    #filename = "races.csv"
+    #f = open(filename, "w")
+
+    page = str(p)
+
+    my_url = 'https://ceskybeh.cz/terminovka/?region=0&dfrom=01.%2001.%202019&dto=31.%2003.%202021&rlength=0&rtype=0&advanced=1&search=&page=' + page
+
+    uClient = uReq(my_url)
+    page_html = uClient.read()
+    uClient.close()
+    page_soup = soup(page_html, "html.parser")
+
+
+    # This function find list with 35 items, which have selector/locator "class = row".
```

Comment:
Note that "locator" is a terminology from the project you work on at work, the proper term here indeed is "selector".

Reply:
deleted